packages/plugin-lighthouse/src/lib/runner/constants.unit.test.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'vitest';
+import { auditSchema, groupSchema } from '@code-pushup/models';
+import { LIGHTHOUSE_GROUPS, LIGHTHOUSE_NAVIGATION_AUDITS } from './constants';
+
+describe('constants', () => {
+  it.each(LIGHTHOUSE_NAVIGATION_AUDITS.map(a => [a.slug, a]))(
+    'should parse audit "%s" correctly',
+    (slug, audit) => {
+      expect(() => auditSchema.parse(audit)).not.toThrow();
+      expect(audit.slug).toEqual(slug);
+    },
+  );
+
+  it('should only include audits supporting the "navigation" mode', () => {
+    const invalidAuditSlugs = new Set([
+      'interaction-to-next-paint',
+      'uses-responsive-images-snapshot',
+      'work-during-interaction',
+    ]);
+    expect(
+      LIGHTHOUSE_NAVIGATION_AUDITS.some(({ slug }) =>
+        invalidAuditSlugs.has(slug),
+      ),
+    ).toBe(false);
+  });
```

Comment:
🧪 One use case per test: Separate invalid audits should be tested separately. These values can be added to the test description via dynamic testing.
```suggestion
    it.each([
    'interaction-to-next-paint',
    'uses-responsive-images-snapshot',
    'work-during-interaction',
  ])(
    'should not include audit %s which does not support the "navigation" mode',
    invalidSlug => {
      expect(
        LIGHTHOUSE_NAVIGATION_AUDITS.every(({ slug }) => slug !== invalidSlug),
      ).toBe(true);
    },
  );
```